unordered_associative_task_1/student/include/impl.h

```diff
@@ -32,4 +33,21 @@ using playlist = std::vector<Song>;
  *
  * @return true if first playlist is anagram of second
  */
-bool is_same_content(const playlist& first_playlist, const playlist& second_playlist);
+bool is_same_content(const playlist& first_playlist, const playlist& second_playlist){
+    
+    if(first_playlist.size() != second_playlist.size()){
+        return false;
+    }
+    
+    std::unordered_multiset<std::string> playlist1 {}, playlist2 {};
+
+    for( const auto& s : first_playlist){
+        playlist1.insert(s.name());
+    }
+
+    for( const auto& s : second_playlist){
+        playlist2.insert(s.name());
+    }
+    
+    return playlist1 == playlist2;
```

Comment:
so what is the profit of using unordered_multiset here? what is the difference with plain vector?